### ISSUE TITLE
Invalidate LyraConfig on git pull

### ISSUE THREAD

**Issue by WULCAN** [zetkin/lyra#84]
Lyra uses git pull during and after a pull-request is created, but it does not re-read LyraConfig after that. If the list of allowed languages changed in the remote repository, or if the location or formats of messages and translations changed, we would probably have problems until Lyra server is restarted.

See `RepoGit.getLyraConfig` and `api/pull-request/[projectName]/route.ts`

**Comment by a-jaxell** [comment]
> api/pull-request/[projectName]/route.ts 

I guess this is outdated since we are using server actions now.  Does this for this issue in general aswell?

**Comment by WULCAN** [comment]
Yes, the API route is outdated, but the server action inherited the same issue. It pulls but does not re-read LyraConfig.

https://github.com/zetkin/lyra/blob/cf9907e6e6ea6049b5c073e076039f57397d3ee4/webapp/src/actions/createPullRequest.ts#L61-L65

https://github.com/zetkin/lyra/blob/cf9907e6e6ea6049b5c073e076039f57397d3ee4/webapp/src/RepoGit.ts#L83-L84

https://github.com/zetkin/lyra/blob/cf9907e6e6ea6049b5c073e076039f57397d3ee4/webapp/src/RepoGit.ts#L148-L154
